src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/CatalogPromotion.xml

```diff
@@ -129,6 +129,8 @@ Example configuration for `percentage_discount` action type:
             </attribute>
         </property>
         <property name="rules" writable="true" />
-        <property name="actions" writable="true" />
+        <property name="actions" writable="true">
+            <subresource resourceClass="%sylius.model.catalog_promotion_action.class%" collection="true" />
```

Comment:
we don't want to have it as a subresource, because actions (and rules) make sense only with catalog promotion.
So IMO we should find some different solution.